include/drivers/led.h

```diff
@@ -22,6 +22,10 @@
 #include <zephyr/types.h>
```

Comment:
Please change the commit subject to `include: drivers: add missing extern "C" in led.h and lora.h`

You need to edit your local commit for this, and add the signed-off-by line there, and then force push your changes.

Reply:
Please fix the commit subject in the **commit** itself, the changes in github PR do not get propagated into the actual commit